### PR TITLE
CocoaPods support

### PR DESCRIPTION
**Pull request by jcavar**
This adds an easy way to integrate Mosquitto library within iOS  project using CocoaPods

### CODE REVIEW

**Comment by ralight** [comment]
This looks fine, but could you please sign the Eclipse CLA and redo the commit with a "Signed-Off" line. You can do this when you commit by using "git commit -s"

File: Mosquitto.podspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Pod::Spec.new do |s|
+  s.name             = "Mosquitto"
```
Comment (kartben) [inline Mosquitto.podspec]: Eclipse Mosquitto

File: Mosquitto.podspec [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  s.version          = "1.4.8"
+  s.summary          = "Mosquitto is an open source implementation of a server for version 3.1 and 3.1.1 of the MQTT protocol."
```
Comment (kartben) [inline Mosquitto.podspec]: Eclipse Mosquitto

File: Mosquitto.podspec [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  s.description      = "Mosquitto is an open source implementation of a server for version 3.1 and 3.1.1 of the MQTT protocol."
```
Comment (kartben) [inline Mosquitto.podspec]: Eclipse Mosquitto :)

**Pull request closed without merging**